constants/labels.js

```diff
@@ -0,0 +1,65 @@
+
+const LABEL_GROUPS = {
+    CUSTOMER: 'Customer',
+    BUG: 'Bug',
+    FEATURE: 'Feature'
+};
+
+const LABEL_TYPES_BUG = {
+    PRODUCTION: 'production',
+    DEVELOPMENT: 'development'
+}
+
+const LABEL_TYPES_FEATURE = {
+    NEW: 'new',
+    EXISTING: 'existing'
+}
+
+const parseLabel = label => {
+  if (!label) return { group: null, type: null };
+  const [group = '', type= ''] = label.split(';');
+  return {
+    group: group.trimRight(),
+    type: type.trimLeft(),
+  };
+}
+
+function getTitle (label) {
+    const { group, type } = parseLabel(label);
+
+    switch(prefix) {
```

Comment:
should this be:

```suggestion
    switch(label) {
```

and then the `prefix` below be changed as well?